Gothic-UnZENity-Project/Gothic-UnZENity
# Feature/lfs removal

# To test
1. Start Unity once
2. Load game to boot world.scene (FeatureFlag.SkipMainMenu is fine)
3. Please test also with an Android build. I checked the .meta files of the .dll's/.so's about when they should be baked into an MVR release, but please double check if it works correctly.

Changes:
* Moved .dll dependencies (ZenKit etc.) into package (Already set, Unity will download once Editor is opened)
* Moved world scenes into same package (Already linked inside Build Settings; therefore world.zen will load)
* Removed DLLs for FilePickerButton
